terraform/deployments/cluster-services/argo.tf

```diff
@@ -76,76 +78,45 @@ resource "helm_release" "argo_notifications" {
+resource "helm_release" "argo_workflows" {
+  chart      = "argo-workflows"
+  name       = "argo-workflows"
+  namespace  = local.services_ns
+  repository = "https://argoproj.github.io/argo-helm"
+  version    = "0.8.0" # TODO: Dependabot or equivalent so this doesn't get neglected.
+  values = [yamlencode({
+    controller = {
+      workflowNamespaces = [local.services_ns, local.apps_ns]
     }
 
-    triggers = {
-      "trigger.sync-operation-change" = yamlencode([
-        {
-          when    = "app.status.operationState.phase in ['Succeeded'] and app.status.health.status == 'Healthy'"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-        {
-          when    = "app.status.operationState.phase in ['Running']"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-        {
-          when    = "app.status.operationState.phase in ['Error', 'Failed']"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-      ])
+    workflow = {
+      serviceAccount = {
+        create = true # Specifies whether a service account should be created
+      }
     }
+  })]
+}
 
-    templates = {
-      "template.send-slack" = yamlencode({
-        webhook = {
-          slack_webhook = {
-            method = "POST"
-            body   = <<-EOB
-            {
-              "attachments": [{
-                "title": "{{.app.metadata.name}}",
-                "title_link": "{{.context.argocdUrl}}/applications/{{.app.metadata.name}}",
-                {{- if eq .app.status.operationState.phase "Succeeded" -}}
-                "color": "#18be52",
-                {{- else if eq .app.status.operationState.phase "Running" -}}
-                "color": "#beb618",
-                {{- else if or (eq .app.status.operationState.phase "Error") (eq .app.status.operationState.phase "Failed") -}}
-                "color": "#be1b18",
-                {{- else -}}
-                "color": "#183cbe",
-                {{- end -}}
-                "fields": [{
-                  "title": "state",
-                  "value": "{{.app.status.operationState.phase}}",
-                  "short": true
-                }]
-              }]
-            }
-            EOB
-          }
-        }
-      })
-    }
+resource "helm_release" "argo_events" {
+  chart      = "argo-events"
+  name       = "argo-events"
+  namespace  = local.services_ns
+  repository = "https://argoproj.github.io/argo-helm"
+  version    = "1.7.0" # TODO: Dependabot or equivalent so this doesn't get neglected.
+  values = [yamlencode({
+    namespace                        = local.services_ns
+    "sensorController.extraEnv"      = [{ "name" : "DEBUG_LOG", "value" : "true" }]
+    "eventsourceController.extraEnv" = [{ "name" : "DEBUG_LOG", "value" : "true" }]
+    "eventbusController.extraEnv"    = [{ "name" : "DEBUG_LOG", "value" : "true" }]
```

Comment:
Don't forget to remove these - you mentioned you didn't intend to leave them in.